Azure/windows-webapp-template.json

```diff
@@ -31,18 +28,13 @@
                     "name": "Microsoft.ApplicationInsights.AzureWebSites",
                     "apiVersion": "2015-08-01",
                     "dependsOn": [
```

Comment:
you can remove this entire resources [] block as it is installing appinsights extension. (Sorry for missing this earlier)

Reply:
fixed by removing the resources